packages/core/src/shaderlib/pbr/util_frag_define.glsl

```diff
@@ -57,12 +57,6 @@ float punctualLightIntensityToIrradianceFactor( const in float lightDistance, co
 
 }
 
-vec3 BRDF_Diffuse_Lambert( const in vec3 diffuseColor ) {
-
-	return RECIPROCAL_PI * diffuseColor;
-
-}
-
```

Comment:
please don't delete it. lambert coefficient is used to convert irradiance to radiance in both global illumination